workers/unity/Packages/io.improbable.gdk.gameobjectcreation/Tests/Editmode/GameObjectCreationTests.cs

```diff
@@ -0,0 +1,153 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Gdk.Core;
+using Improbable.Gdk.Subscriptions;
+using Improbable.Gdk.TestUtils;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+using Object = UnityEngine.Object;
+
+namespace Improbable.Gdk.GameObjectCreation.EditmodeTests
+{
+    public class GameObjectCreationTests : MockBase
+    {
+        private long entityId = 100;
+
+        private const string WorkerType = "TestWorkerType";
+
+        private EntityGameObjectLinker linker;
+
+        protected override MockWorld.Options GetOptions()
+        {
+            return new MockWorld.Options
+            {
+                AdditionalSystems = new (Type systemType, object[] constructorArgs)[]
+                {
+                    (typeof(GameObjectInitializationSystem),
+                        new object[] { new TestGameObjectCreator(WorkerType), null })
+                }
+            };
+        }
+
+        [SetUp]
+        public new void Setup()
+        {
+            base.Setup();
+
+            var goInitSystem = World.GetSystem<GameObjectInitializationSystem>();
+            linker = goInitSystem.Linker;
+        }
```

Comment:
The `AdditionalSystems` change in `MockWorld` was done to simplify the setup of these tests